pkg/controllers/multicluster/remotevtep_controller.go

```diff
@@ -76,21 +76,23 @@ func (r *RemoteVtepReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 	}()
 
-	var node = &corev1.Node{}
-	if err = r.Get(ctx, req.NamespacedName, node); err != nil {
+	var nodeInfo = &networkingv1.NodeInfo{}
+	if err = r.Get(ctx, req.NamespacedName, nodeInfo); err != nil {
 		if apierrors.IsNotFound(err) {
 			return ctrl.Result{}, wrapError("unable to clean VTEP for node", r.cleanVTEPForNode(ctx, req.Name))
 		}
 		return ctrl.Result{}, wrapError("unable to get node", err)
 	}
 
-	if !node.DeletionTimestamp.IsZero() {
+	if !nodeInfo.DeletionTimestamp.IsZero() {
 		log.V(1).Info("ignore terminating node")
 		_ = r.cleanVTEPForNode(ctx, req.Name)
 		return ctrl.Result{}, nil
 	}
 
-	var vtepIP, vtepMac, vtepVxlanIPList = node.Annotations[constants.AnnotationNodeVtepIP], node.Annotations[constants.AnnotationNodeVtepMac], node.Annotations[constants.AnnotationNodeLocalVxlanIPList]
+	var vtepIP, vtepMac, vtepVxlanIPList = nodeInfo.Spec.VTEPInfo.IP, nodeInfo.Spec.VTEPInfo.MAC,
```

Comment:
Should `nodeInfo.Spec.VTEPInfo` be checked whether to be nil here?

Reply:
The VTEPInfo field for RemoteVtep object is inline and required, so it might be unnecessary.

Reply:
Yes, in `RemoteVtep`, it is value type, but in `NodeInfo`, it is pointer type.

Reply:
done